op-batcher/batcher/channel_manager_test.go

```diff
@@ -199,29 +192,41 @@ func TestClearChannelManager(t *testing.T) {
 	require.Empty(t, m.confirmedTransactions)
 
 	// Add a block to the channel manager
-	a, _ := derivetest.RandomL2Block(rng, 4)
+	lBlock := types.NewBlock(&types.Header{
```

Comment:
why was this changed?

Reply:
Ah it broke because the randomness would allow the block to be too small